lib/compile.js

```diff
@@ -130,7 +131,7 @@ var Compiler = (function () {
                     msgid = msgid.replace( unconvertedEntityPattern, convertedEntity );
                 }
 
-                if ( removeWhiteSpaces ) {
+                if ( removeWhiteSpaces && !isDBString ) {
```

Comment:
that's just how it was in angular-gettext, it's not our library, we just forked it and sticking to the same silly layout like they have :)